src/Fl_Group.cxx

```diff
@@ -487,6 +555,22 @@ void Fl_Group::insert(Fl_Widget &o, int index) {
 */
 void Fl_Group::add(Fl_Widget &o) {insert(o, children_);}
 
+/**
+ Allow derived groups to act when a child widget is removed from the group.
+
+ Widgets derived from Fl_Group may store additional data for their children.
+ Overriding this method will allow derived classes to remove these data
+ structures just before the child is removed.
+
+ \note if children are removed by Fl_Group::clear(), this method is still
+ called, but the original indices may be reversed.
```

Comment:
This \note is obsolete since my recent commit.

Reply:
I guess you have not done a "git pull" in like forever, or 3 minutes ;-)

Reply:
Indeed. Pulled from the new branch meanwhile.